card/spec/lib/card/set_pattern_spec.rb

```diff
@@ -44,13 +44,13 @@ def it_generates( opts )
 end
 
 describe Card::AllPlusSet do
-  it_generates :name => "*all plus", :from => Card.new( :name => "Book+author" )
+  it_generates name: "*all plus", from: Card.new( name: "Book+author" )
```

Comment:
Space inside parentheses detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.